jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/ArrayByteBufferPool.java

```diff
@@ -719,38 +720,104 @@ protected void acquire()
         }
     }
 
+    /**
+     * A variant of the {@link ArrayByteBufferPool} that
+     * uses a predefined set of buckets of buffers.
+     */
+    public static class Predefined extends ArrayByteBufferPool
```

Comment:
Don't like "predefined", it hints to me that I have no control, while it's the opposite.

How about `Custom` or `CustomBuckets` or `WithBucketCapacities`?

Reply:
I went with `WithBucketCapacities`.